sensor_msgs/Image bgr
sensor_msgs/Image mask
string save_name
---
bool success
Int32 label
string msg